server/broadcast.go

```diff
@@ -56,8 +56,19 @@ var submitMultiSession = func(ctx context.Context, sess *BroadcastSession, seg *
 var maxTranscodeAttempts = errors.New("hit max transcode attempts")
 
 type BroadcastConfig struct {
-	maxPrice *core.AutoConvertedPrice
-	mu       sync.RWMutex
+	maxPricePerCapability map[core.Capability]map[string]*core.AutoConvertedPrice
+	mu                    sync.RWMutex
+}
+
+func newBroadcastConfig() *BroadcastConfig {
+	models := make(map[string]*core.AutoConvertedPrice)
+	models["default"] = core.NewFixedPrice(big.NewRat(0, 1))
+
+	return &BroadcastConfig{
+		maxPricePerCapability: map[core.Capability]map[string]*core.AutoConvertedPrice{
+			core.Capability_H264: models,
```

Comment:
As discussed in our chat this can be core.Capability_Unused.

Reply:
Updated